### PR TITLE
cpp-language-construct-add

### PR DESCRIPTION
**Pull request by aayush105**

### CODE REVIEW

File: CppLanguageConstructs/q12.cpp
Comment (thegauravgiri) [inline CppLanguageConstructs/q12.cpp:1]: Another simple code for this question as there is already a swapping question.

File: CppLanguageConstructs/q16.cpp
Comment (thegauravgiri) [inline CppLanguageConstructs/q16.cpp:1]: Delete this.

File: CppLanguageConstructs/q5.cpp [hunk 1]
```diff
@@ -0,0 +1,43 @@
+//  WAP to calculate area of circle(radius), rectangle(l,b), area of triangle(a,b,c) using function overloading
+
+#include <iostream>
+#include <cmath>
+using namespace std;
+
+// Function to calculate area of a circle
+float area(float radius) {
+    return 3.14 * radius * radius;
```
Comment (thegauravgiri) [inline CppLanguageConstructs/q5.cpp]: use #define macro for 3.14 set it to PI

**Review by thegauravgiri — CHANGES_REQUESTED** [review]

File: CppLanguageConstructs/q12.cpp
Comment (aayush105) [reply CppLanguageConstructs/q12.cpp:1]: ok got it

File: CppLanguageConstructs/q16.cpp
Comment (aayush105) [reply CppLanguageConstructs/q16.cpp:1]: sureee

File: CppLanguageConstructs/q5.cpp (hunk 1 above)
Comment (aayush105) [reply CppLanguageConstructs/q5.cpp]: okay suree

**Comment by aayush105** [comment]
Check it bro @gaurovgiri ,Ii have updated the solution of the problem that you had mentioned.

**Review by thegauravgiri — APPROVED** [review]
LGTM

**Pull request merged**